src/cdk-experimental/listbox/listbox.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive} from '@angular/core';
+
+@Directive({
+    selector: '[cdkListboxOption]',
+    exportAs: 'cdkListboxOption',
+    host: {
+        role: 'option',
+    }
+})
+export class CdkListboxOption {
+
+}
+
+@Directive({
+    selector: '[cdkListbox]',
+    exportAs: 'cdkListbox',
+    host: {
+        role: 'listbox',
+        tabindex: '0'
```

Comment:
I would leave off the tabindex and deal with focus management in a follow-up PR focused just on dealing with focus/activedescendant